tests/searchSpec.js

```diff
@@ -195,6 +195,39 @@ describe('search', function() {
     done();
   });
 
+  it('makes search with non existing filter value with conjunction true should return no results', function test(done) {
+
+    const itemsjs = require('./../index')(items, configuration);
+    
+    const result = itemsjs.search({
+      filters: {
+        category: ['drama', 'thriller']
+      }
+    });
+    
+    assert.equal(result.data.items.length, 0);
+    assert.equal(result.data.aggregations.tags.buckets[0].doc_count, 0);
+    
+    done();
+  });
+  
+  it('makes search with non existing filter value with conjunction false should return results', function test(done) {
+    
+    configuration.aggregations.category.conjunction = false;
```

Comment:
This test modifies a shared configuration value (`configuration.aggregations.category.conjunction`). For test independence, please consider:

1. Using a dedicated configuration copy just for this test, or
2. Resetting the configuration to its initial state post-test.

This approach ensures tests don't inadvertently influence each other.